src/datatypes/CardPackage.ts

```diff
@@ -1,16 +1,27 @@
 import { CardDetails } from './Card';
+import User from './User';
 
-export const APPROVED = 'a' as const;
-export const SUBMITTED = 's' as const;
+export enum CardPackageStatus {
+  APPROVED = 'a',
+  SUBMITTED = 's',
+}
 
-export default interface CardPackage {
-  id: string;
+export type UnhydratedCardPackage = {
+  id?: string;
   title: string;
-  date: string;
+  date: number;
   owner: string;
```

Comment:
Before owner was typed only as a string even though hydration clearly set it as a User